radar-auth/src/main/java/org/radarcns/auth/token/JwtRadarToken.java

```diff
@@ -117,45 +115,26 @@ public String getType() {
     }
 
     private Map<String, List<String>> parseRoles(DecodedJWT jwt) {
-        if (!jwt.getClaims().containsKey(ROLES_CLAIM)) {
-            return Collections.emptyMap();
-        }
-
-        Map<String, List<String>> result = new HashMap<>();
-        List<String[]> parsedRoles = jwt.getClaim(ROLES_CLAIM).asList(String.class).stream()
+        return emptyIfNull(jwt.getClaim(ROLES_CLAIM).asList(String.class)).stream()
```

Comment:
Would it return an empty map? I only see methods to return empty string or list.

Reply:
It would process the empty list and finally return an empty map after the collect method.